app/javascript/src/modules/m3_viewer.js

```diff
@@ -20,7 +20,8 @@ export default {
     const data = el.dataset
     const showAttribution = (data.showAttribution === 'true')
     const hideWindowTitle = (data.hideTitle === 'true')
-    const imageTools = (data.imageTools === 'true')
+    // TODO: Remove deprecated data.imageTools once exhibits is updated to use enableComparison
+    const enableComparison = (data.enableComparison === 'true' || data.imageTools === 'true')
```

Comment:
I don't think you need data.imageTools here. I am pretty sure there data fields come from the data fields and you changed `data-image-tools` to `data-enable-comparison='<%= @embed_request.enable_comparison? %>'`

Reply:
I think you're right. For whatever reason I thought exhibits was supplying its own data attributes, but it's just supplying the parameter.

Fixed.